layout/Speakers/components/Schedule/Table/Block/index.jsx

```diff
@@ -91,8 +91,15 @@ export default function Block({
           </div>
         </div>
       </div>
-
-      <p className={showSpeaker ? "hidden" : ""}>{description}</p>
+      <div>
+        <p
+          className={`transition-max-height overflow-hidden duration-300 max-h-${
```

Comment:
```suggestion
          className={`transition-max-height overflow-hidden duration-300 ${showSpeaker ? "max-h-96" : "max-h-0"}`}
```

Reason: tailwind analyses all source files to find css classes to avoid outputting the whole library. By splitting up the css class name, this analysis won't find it. The only reason why the animation still works in this case is because this exact same class is found in another source file (in the schedule, where it's used for the exact same animation)